monitoring/environments/civo-demo/main.jsonnet

```diff
@@ -129,6 +129,12 @@ local prometheuses = [
         query+: {
           lookbackDelta: '15m',  // Analytics are only sent once every 10m
         },
+        thanos: {
+          objectStorageConfig: {
+            name: 'parca-analytics-objectstorage',
+            key: 'thanos.yaml',
+          },
```

Comment:
This needs to go in the function `params` after line 35.

Could you add resources limits/requests too, please?

```jsonnet
          resources+: {
            limits+: {
              memory: '64Mi',
            },
            requests+: {
              cpu: '100m',
              memory: '64Mi',
            },
          },
```